appserver/admingui/common/src/main/java/org/glassfish/admingui/common/handlers/CommonHandlers.java

```diff
@@ -846,10 +845,9 @@ public static List filterAdminObjects(HandlerContext context) {
     private static String handleBareAttribute(FacesContext ctx, String url) {
 	// Get Page Session...
 	UIViewRoot root = ctx.getViewRoot();
-	Map<String, Serializable> pageSession =
-	    PageSessionResolver.getPageSession(ctx, root);
+	Map<String, Serializable> pageSession = null;
```

Comment:
```suggestion
	Map<String, Serializable> pageSession = createPageSession(ctx, root);
```

Reply:
you can see this change because of the base branch Payara6 that I only used to execute the jenkins pipeline. The correct base branch should be the Payara6-tck that is not showing this change